_includes/v23.1/ui/insights.md

```diff
@@ -26,18 +26,23 @@ Transaction executions with the **High Contention** insight are transactions tha
 {% if page.cloud != true -%}
 The following screenshot shows the execution of a transaction flagged with **High Contention**:
 
-<img src="{{ 'images/v22.2/transaction_execution.png' | relative_url }}" alt="Transaction execution" style="border:1px solid #eee;max-width:100%" />
+<img src="{{ 'images/v23.1/transaction_execution.png' | relative_url }}" alt="Transaction execution" style="border:1px solid #eee;max-width:100%" />
 {% endif -%}
 
 To view [details of the execution](#transaction-execution-details), click an execution ID in the **Latest Transaction Execution ID** column.
 
 - **Latest Transaction Execution ID**: The execution ID of the latest execution with the transaction fingerprint.
 - **Transaction Fingerprint ID**: The transaction fingerprint ID of the latest transaction execution.
 - **Transaction Execution**: The transaction fingerprint of the latest transaction execution.
+- **Status**: The transaction status.
 - **Insights**: The insight for the transaction execution.
   - **High Contention**: The transaction execution experienced high [contention]({{ link_prefix }}performance-best-practices-overview.html#transaction-contention) time according to the threshold set in the [`sql.insights.latency_threshold` cluster setting]({{ link_prefix }}cluster-settings.html#setting-sql-insights-latency-threshold).
+  - **Suboptimal Plan**: The plan could be improved for some statement(s) in the transaction. Possible causes include outdated statistics and missing indexes.
+  - **Slow Execution**: The transaction took longer than the value of the [`sql.insights.latency_threshold` cluster setting]({{ link_prefix }}cluster-settings.html#setting-sql-insights-latency-threshold) to execute.
```

Comment:
Similar to Suboptimal plan. This insight is statement-level but aggregated up to the transaction. 

The transaction had one or more statements that took longer than the value of the ...